eng/build.ps1

```diff
@@ -312,7 +311,7 @@ foreach ($argument in $PSBoundParameters.Keys)
     "hostConfiguration"      { $arguments += " /p:HostConfiguration=$((Get-Culture).TextInfo.ToTitleCase($($PSBoundParameters[$argument])))" }
     "framework"              { $arguments += " /p:BuildTargetFramework=$($PSBoundParameters[$argument].ToLowerInvariant())" }
     "os"                     { $arguments += " /p:TargetOS=$($PSBoundParameters[$argument])" }
-    "allconfigurations"      { $arguments += " /p:BuildAllConfigurations=true" }
+    "pack"                   { $arguments += " -pack /p:BuildAllConfigurations=true" }
```

Comment:
Any particular reason we're keeping around the `BuildAllConfigurations=true` setting? Is the `Pack=true` setting not a good marker for our build in this case? Is this for the runtimelab scenario?

Reply:
The Pack property doesn't flow into the repo build as Arcade's Build.proj unsets it.

Overall we want to keep the BuildAllConfigurations property around to allow building all TFMs without generating packages (for runtimelab and potentially in dev innerloop).